backend/app/routes/broadcast/getBroadcasts.js

```diff
@@ -0,0 +1,57 @@
+const to = require('await-to-js').default;
+const Broadcast = require('../../models/Broadcast');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+const limit = constants.PAGINATION_LIMIT.GET_BROADCASTS;
+
+const getBroadcastsAggregate = (startIndex, match) => {
+  const pipeline = [
+    { $match: match },
+    {
+      $project: {
+        _id: 1,
+        title: 1,
+        // imageUrl: 1,
+        // link: 1,
+        // content: 1,
+        // tags: 1,
+        // expiresOn: 1,
+        createdAt: 1,
+        // updatedAt: 1,
+      },
+    },
+    { $sort: { createdAt: -1 } },
+    { $skip: startIndex && startIndex - 1 },
+    { $limit: startIndex ? limit + 2 : limit + 1 },
+  ];
+  return pipeline;
+};
+module.exports = async (req, res, next) => {
+  const page = parseInt(req.query.page, 10) || 1;
+  // match for tag filtering
+  const match = req.query.tags ? { tags: { $in: req.query.tags.split(',') } } : {};
+  const startIndex = (page - 1) * limit;
+  const results = {};
+  const [err, broadcasts] = await to(Broadcast.aggregate(getBroadcastsAggregate(startIndex, match)).exec());
+  if (err) {
+    const error = new ErrorHandler(constants.ERRORS.DATABASE, {
+      statusCode: 500,
+      message: 'Mongo Error: Fetching Failed',
+      errStack: err,
+    });
+    return next(error);
+  }
+  // check if previous page exists
+  if (startIndex > 0) {
+    [results.previous] = broadcasts;
```

Comment:
In frontend previous and next can be used to show 
1.navigation if there is a previous page or next page
2.the next and previous post's title can be used to show what's on the next page near the next button.

If you want it some other way please mention, I will change according to your need.

Reply:
you mean in the carousel?

Reply:
https://hitk-tech-community.netlify.app/broadcasts
check this, I dont think so there is any scope to show half broadcast in the carousel effect (something like this https://webartdevelopers.com/blog/wp-content/uploads/2020/07/3d-carousel-slider-vanilla-js.gif to show half left and right in the effect

Reply:
Also, I suggest you to deep dive on pagination, I checked the source code and ik where you picked it from, but personally I do not think so that article was a good implementation. Check out pagination methods and issues with pagination wrt a particular method

Reply:
ok....so what should i do. 
1.remove next and previous
2.add filters for month and year

Reply:
??

Reply:
@jackfrost13 please reply so that I can proceed.

Reply:
> ok....so what should i do. 
> 1.remove next and previous
> 2.add filters for month and year

Yes do this,
Sorry Couldn't reply, down with covid